src/cpp/session/modules/SessionConnectionsInstaller.R

```diff
@@ -454,6 +454,7 @@
 
    # Find driver.ini file
    driverIniFile <- .rs.odbcBundleDriverIniPath(name, installPath)
+   driverIniFile <- .rs.odbcBundleDriverIniPath(installPath)
```

Comment:
This looks like a copy-paste. I think you didn't want the line above, since it still has `name` in the call.

Reply:
Thanks for catching this!